infra/fastly/terraform/dl-sandbox.k8s.dev/vcl/binaries.vcl

```diff
@@ -45,7 +45,7 @@ sub vcl_fetch {
   }
 
   # Ensure version markers are not cached at the edge
-  if (req.url.path ~ "^/release/(latest|stable)(-\d+(\.\d+))?\.txt\z") {
+  if (req.url.path ~ "^/release/(latest|stable)(-\d+(\.\d+)?)?\.txt\z") {
```

Comment:
Why don't we do:
```suggestion
  if (req.url.path ~ "^/release/(latest|stable)([^/]*)\.txt\z") {
```

So any top level release file that starts with latest or stable, ends with .txt

Reply:
@BenTheElder SGTM. PTAL.